apollo-router/Cargo.toml

```
@@ -126,6 +127,7 @@ notify = { version = "6.1.1", default-features = false, features = [
 ] }
 nu-ansi-term = "0.50"
 num-traits = "0.2.19"
+oci-client = { version = "0.14.0", default-features = false, features = ["rustls-tls", "rustls-tls-native-roots", "trust-dns", "test-registry"] }
```

We don't need `test-registry`, I'll remove that from `towering` as well.

```suggestion
oci-client = { version = "0.14.0", default-features = false, features = ["rustls-tls", "rustls-tls-native-roots", "trust-dns"] }
```